test/integration/change-streams/change_stream.test.ts

```diff
@@ -1990,4 +1992,99 @@ describe('Change Streams', function () {
         .toJSON()
     )
     .run();
+
+  describe('BSON Options', function () {
+    let client: MongoClient;
+    let db: Db;
+    let collection: Collection;
+    let cs: ChangeStream;
+    beforeEach(async function () {
+      client = await this.configuration
+        .newClient({ maxPoolSize: 1, monitorCommands: true })
+        .connect();
+      db = client.db('db');
+      collection = db.collection('collection');
+    });
+    afterEach(async function () {
+      await cs.close();
+      await client.close();
+      client = undefined;
+      db = undefined;
+      collection = undefined;
+    });
+    it('Should remain type long', async function () {
```

Comment:
Oh one other note - change streams are only available on replica sets or sharded topologies.  Your tests are failing in CI (something is wrong with them) but the server tests will fail regardless, since change streams aren't supported. 

We have the ability to filter for specific topologies (or other conditions) for our tests.  There are other examples in this particular file (look for metadata).  We want to run these tests on all topologies _except_ single servers so we'll need to add a metadata filter for that.